<launch>
    <arg name="motor_poles"/>
    <arg name="gear_ratio"/>
    <arg name="continuous_current_limit"/>

    <node name="socketcan_bridge" pkg="socketcan_bridge" type="socketcan_bridge_node"/>

    <node name="karelics_vesc_can_driver" pkg="karelics_vesc_can_driver" type="vesc_can_driver.py" output="screen">
        <param name="motor_poles" value="$(arg motor_poles)"/>
        <param name="gear_ratio" value="$(arg gear_ratio)"/>
        <param name="continuous_current_limit" value="$(arg continuous_current_limit)"/>
    </node>

</launch>